app/theme/client/imports/components/header.css

```diff
@@ -45,7 +42,7 @@
 	&--room {
 		padding: 1.25rem;
 
-		box-shadow: 0 1px 2px 0 rgba(31, 35, 41, 0.08);
+		border-bottom: 2px solid #F7F8FA;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#F7F8FA" to be "#f7f8fa" (color-hex-case)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5416119)